ducktape/tests/test.py

```diff
@@ -27,6 +27,7 @@
 from ducktape.command_line.defaults import ConsoleDefaults
 from ducktape.services.service_registry import ServiceRegistry
 from ducktape.template import TemplateRenderer
+from ducktape.mark.resource import CLUSTER_SPEC_KEYWORD
```

Comment:
nit: you can do `from ducktape.mark.resource import CLUSTER_SIZE_KEYWORD, CLUSTER_SPEC_KEYWORD` here. generally it's nicer to group imports from the same module where possible

Reply:
Done!